### DISCUSSION TITLE
YouTube has playing issue after login.

### DISCUSSION THREAD

**Discussion by hugo-sama** [Eloston/ungoogled-chromium#3070] [Q&A]
recently I got playback issue with youtube, it keep saying Something went wrong. Refresh or try again later.
[Learn more](https://support.google.com/youtube?p=videoError&hl=en).
It might play for the first couple seconds and wont load more video after the webpage got loaded, and it will show this error no matter u seek or change the resolution. Its weird that this problem only happens after you login your account on youtube. The playback is fine if you not login youtube or using incognito. 
And its fine play youtube after login with same extensions enabled on microsoft edge and firefox. 

Ungoogled Chromium Version: `129.0.6668.100`
Windows 10 Version 22H2 (Build 19045.5011)

**Comment by PF4Public** [comment]
Please do search through [existing issues](https://github.com/ungoogled-software/ungoogled-chromium/issues?q=is%3Aissue+YouTube) first.

**Comment by hugo-sama** [comment]
ahh i thought it was profile corruption or something that happens on user side, so I posted this to discussion and didnt go through the issue list. Thanks for your reply.
